foubus.py

```diff
@@ -379,7 +395,8 @@ def term_write(s):
         term.write(curses.tparm(curses.tigetstr('sgr'), 0))
         term_write('\n')
     html.write(f"<div>Times include walking time to the stop.</div>\n")
-    html.write(f"<div>Last updated: {now}</div>\n")
+    html.write(f'<div>Last updated: <span class="last-updated-time">00:00</span> ago</div>\n')
```

Comment:
What if the presentation machine freezes? Will this become stale?

Reply:
Good point.

Reply:
Thanks, I see the change you made for this.

But once the client-side JavaScript (updateCountdowns) updates the line once, then if the machine freezes, we're back to displaying misleading information.

Reply:
So how about dropping the whole lastUpdatedDiv thing?

Reply:
working on a solution rn, testing what looks good

Reply:
I'm not super attached to it, I was mostly annoyed by the size and illegibility of the absolute time, which is fixed now. I'll keep the relative time to see how it is, but it's probably not necessary and might even be distracting.
By the way, this whole pr was unnecessary, it was only not updating because you directly put the schedule.html into homeassistant, instead of the whole doc that updates an iframe with the schedule.

Reply:
Oh, my bad. I don't know why I would have done that. Thanks for tracking it down.

What's your intent with this PR then? still asking to get it merged?

My preference is always "less code" but if you do still want it I'm open to it.